client/js/table-view.js

```diff
@@ -38,8 +38,6 @@ class TableView {
 		this.renderTableHeader();
```

Comment:
Set your Sublime Text preferences to use 2-space tabs instead of 4-space tabs and to “transform indentations to spaces”; this will make all *future* files use 2 spaces instead of 4, which helps readability. 

For all current files (X-Cell Sum), please select “Tab Width: 2” and “Convert indentation to spaces” from the bottom right corner of Sublime Text. Do this for each file, and save each one. Then git commit (make sure that git has registered changes; if it hasn’t, the changed didn’t work) and push changes up to this GitHub branch.